api/api.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	controllers "bootCampApi/api/controllers"
+
+	"github.com/gorilla/mux"
+)
+
+func New() *mux.Router {
+
+	router := mux.NewRouter()
+
+	routes(router)
+	return router
+}
+
+func routes(router *mux.Router) {
+	router.HandleFunc("/", controllers.HealthCheck).Methods("GET")
+	router.HandleFunc("/readCSV", controllers.ReadCSV).Methods("GET")
+	router.HandleFunc("/readCSV/{id:[0-9]+}", controllers.ReadCSV).Methods("GET")
+	router.HandleFunc("/writeCSV/{id:[0-9]+}", controllers.GetAnimeByIdC).Methods("GET")
```

Comment:
Several things here.
- Routes should be descriptive enough to give you an idea what are these about (check the recommendation I added).
- No handler function should take care of more that one responsibility; the fact that are two distinct routes is enough to deduce these are different; even when they perform similar actions, you could create an auxiliary func or method, and use it in each handler func.
- When writing to the CSV, you are not getting anything; you have to use the appropriate HTTP verb for the action.

```suggestion
	router.HandleFunc("/animes", controllers.GetAllAnimes).Methods("GET")
	router.HandleFunc("/animes/{id:[0-9]+}", controllers.GetAnimeByID).Methods("GET")
	router.HandleFunc("/animes/{id:[0-9]+}", controllers.PostAnimeByIDExternalAPI).Methods("POST") // or some other similar name.
```